mojaloop/reporting-hub-bop-settlements-ui/values.yaml

```diff
@@ -0,0 +1,338 @@
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+replicaCount: 1
+
+# Disable command so the defaults defined in the Docker image are used.
+# command:
+  # - sleep
+  # - infinity
+
+
+## Enable diagnostic mode in the deployment
+##
+diagnosticMode:
+  ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+  ##
+  enabled: false
+  ## @param diagnosticMode.command Command to override all containers in the deployment
+  ##
+  command:
+    - npm
+    - start
+  ## @param diagnosticMode.args Args to override all containers in the deployment
+  ##
+  args:
+    - --inspect=0.0.0.0:{{ .Values.diagnosticMode.debug.port }}
+
+  ## @param diagnosticMode.debug config to override all debug information
+  ##
+  debug:
+    internalPort: 9229
+    port: 9229
+
+## @param initContainers Add additional init containers to the %%MAIN_CONTAINER_NAME%% pod(s)
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+## e.g:
+## initContainers:
+##  - name: your-image-name
+##    image: your-image
+##    imagePullPolicy: Always
+##    command: ['sh', '-c', 'echo "hello world"']
+##
+initContainers: []
+# initContainers:
+#   - name: wait-for-keto
+#     image: some-image-for-waiting-kafka
+#     imagePullPolicy: IfNotPresent
+#     command:
+#       - sh
+#       - -c
+#       - until ./bin/kafka-broker-api-versions.sh --bootstrap-server ${KAFKA_HOST}:${KAFKA_PORT};
+#         do
+#           echo --------------------;
+#           echo Waiting for Kafka...;
+#           sleep 2;
+#         done;
+#         echo ====================;
+#         echo Kafka ok!;
+#     env:
+#       - name: KAFKA_HOST
+#         value: '{{ include "common.backends.centralbroker.host" . }}'
+#       - name: KAFKA_PORT
+#         value: '{{ include "common.backends.centralbroker.port" . }}'
+
+
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /
+  initialDelaySeconds: 15
+  periodSeconds: 15
+livenessProbe:
+  enabled: true
+  httpGet:
+    path: /
+  initialDelaySeconds: 15
+  periodSeconds: 15
+
+## metric configuration for prometheus instrumentation
+metrics:
+  ## flag to enable/disable the metrics end-points
+  enabled: false
+  config:
+    timeout: 5000
+    prefix: moja_
+    defaultLabels:
+      serviceName: reporting-hub-bop-settlements-ui
+
+config:
+  env:
+    CENTRAL_SETTLEMENTS_ENDPOINT: http://test1-centralsettlement-service:80
+    CENTRAL_LEDGER_ENDPOINT: http://test1-centralledger-service:80
```

Comment:
You can templatize this as we did in the other services.
Ref: https://github.com/mojaloop/charts/blob/ba9234754bddbc64d47b869cea7c1bf96a7fdf5e/mojaloop/role-assignment-service/values.yaml#L356